post-worktime-logger.php

```diff
@@ -222,6 +222,80 @@ function pwlHandleWorktimeReset()
     }
 }
 
+function pwlStatsPage() {
+	if ( !current_user_can( 'manage_options' ) )  {
+		wp_die( __( 'You do not have sufficient permissions to access this page.' ) );
+	}
+	
+	$numOfPosts = 25;
+	
+	//TODO: manage_options users will be able to see complete statistics. is this sensitive data?
+	$args = array(
+		'posts_per_page' => $numOfPosts,
+		'order'		=> 'DESC',
+		'orderby'   => 'meta_value_num',
+		'meta_key'  => 'post-worktime',
+	);
+	
+	$query = new WP_Query( $args );
+	
+	$posts_titles = array();
+	$posts_worktimes = array();
+	
+	if ( $query->have_posts() ) {
+
+		$tPosts = $query->get_posts();
+
+		foreach($tPosts as $tPost){
+			$posts_titles[] = $tPost->post_title;
+			$posts_worktimes[] = get_post_meta($tPost->ID, "post-worktime", true);
```

Comment:
Maybe you should use `round($worktime/3600, 0)`, so we have the hours instead of the seconds.